app/components/quick-filter.js

```diff
@@ -74,7 +74,11 @@ export default class QuickFilter extends Component {
     this.setDateFilter();
     this.set('location', this.dummyLocation);
     this.set('eventName', this.dummyName);
-    this.set('disableClear', false);
+    if (this.dummyName || this.dummyLocation || this.startDate !== null) {
+      this.set('disableClear', false);
+    } else {
+      this.set('disableClear', true);
+    }
```

Comment:
No need of if else, pass the boolean condition as second argument

Reply:
Can you please elaborate? I did not completely understood.

Reply:
Move the condition from if statement to this.set call and remove the if statement

Reply:
Got it.